community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/FailedPopulatingIndexProxyFactory.java

```diff
@@ -57,6 +57,16 @@
     @Override
     public IndexProxy create( Throwable failure )
     {
+        IndexPopulationFailure populationFailure = failure( failure );
+        try
+        {
+            populator.markAsFailed( populationFailure.asString() );
+        }
+        catch ( Exception e )
+        {
+            // we're already in a hole, stop digging
```

Comment:
We could, but there was no logger available in this context. I'll look into injecting one.